client/starwhale/core/runtime/model.py

```diff
@@ -1751,14 +1751,16 @@ def _extract() -> None:
                 self.extract(force=True)
 
         def _render_dockerfile(_manifest: t.Dict[str, t.Any]) -> None:
+            from starwhale.core.runtime.store import get_docker_run_image_by_manifest
```

Comment:
https://github.com/star-whale/starwhale/blob/main/client/starwhale/core/runtime/model.py#L120

This module has already import `starwhale.core.runtime.store`, so move `get_docker_run_image_by_manifest` in the top external import?